llvm/include/llvm/Analysis/DominanceFrontier.h

```diff
@@ -39,7 +39,9 @@ class raw_ostream;
 template <class BlockT, bool IsPostDom>
 class DominanceFrontierBase {
 public:
-  using DomSetType = std::set<BlockT *>;                // Dom set for a bb
+  // Dom set for a bb. Use SetVector to make iterating dom frontiers of a bb
+  // deterministic.
+  using DomSetType = SetVector<BlockT *>;
   using DomSetMapType = std::map<BlockT *, DomSetType>; // Dom set map
```

Comment:
Separately from this PR, I wonder if we should use `DenseMap` here.

Reply:
https://llvm.org/docs/CodingStandards.html says `llvm::DenseMap should almost always be used instead of std::map`, shall I change it to DenseMap?

Reply:
I don't see why not. But I'd open a separate PR for this.

Reply:
thank you all for the review.

> I don't see why not. But I'd open a separate PR for this.

opened a new pr: https://github.com/llvm/llvm-project/pull/70403